packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -235,17 +245,21 @@ const RelationInput = ({
                   }
                   style={style}
                 >
-                  <Box minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
+                  <BoxEllipsis minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
                     {href ? (
-                      <LinkEllipsis to={href} disabled={disabled}>
-                        {mainField ?? id}
-                      </LinkEllipsis>
+                      <Tooltip description={mainField ?? id}>
```

Comment:
Why do you repeat the component instead of having it outside the `href` condition? Then you don't have to do that :)

Reply:
that's what I did first but I hesitated - I thought it would make more sens to attach it directly to the text itself more than a div (which includes a paddingRight (4) so tooltip then appears 16px pixels after the text) 🧐

<img width="276" alt="image" src="https://user-images.githubusercontent.com/71838159/190407242-81961338-281a-431c-9324-7ff68bad3045.png">

Reply:
What I mean is:

```
<BoxEllipsis minWidth={0} paddingTop={1} paddingBottom={1} paddingRight={4}>
  <Tooltip>
  {href ? </> : </>}
  </Tooltip>
</BoxEllipsis>
```

Since the padding is on the Box and not Link or Typography there shouldn't be any padding, no?